salt/states/x509.py

```diff
@@ -715,7 +715,7 @@ def certificate_managed(
             append_file_contents = __salt__["x509.get_pem_entry"](
                 append_file, pem_type="CERTIFICATE"
             )
-            contents += append_file_contents
+            contents += str(append_file_contents)
```

Comment:
I think this will read the PEM file as a byte array, so it will insert it like this:
```
b'-----BEGIN CERTIFICATE-----\nMIIFeDCCA2CgAwIBAgIIbubuNzD0ysgwDQYJKoZIhvcNAQELBQAwKzEpMCcGA1UE\nAwwgYm9vdHN0cmFwLTAubG9jYWwudmlydHVhbGJveC5tYWMwHhcNMjAwOTIzMjEw\nMzE1WhcNMjEwOTIzMjEwMzE1WjArMSkwJwYDVQQDDCBib290c3RyYXAtMC5sb2Nh\nbC52aXJ0dWFsYm94Lm1hYzCCAiIwDQYJKoZIhvcNAQEBBQADggIPADCCAgoCggIB\nAOyfk7y/io45klA98YQjTy4OVwUxUiOXlSCrDHaZwBSOnP2QuJ+EycpLerTOh8yM\nys+3lm2N2kFOVEZH1qAn263cVO0ylVCfVsK+JwWarPhSztaK27or3qrlMu/owhiX\nPqYvZ4n2ppxU3+fDfiQ8sVrf8QmYKJKVs8YHdcgRg5pVpA+hCQ1ZJ0Ls7M8RPCoG\nyurKDOoURqwBBT4/ni6lD7QdzjaqTdzXDjU53z4Uih20kG+et5/stwlbe0A6P1sH\n5DBaWNEL8DsVJ0fhRXARiD4rlxCbP6Q+jMioIYPaKv7ug5tV6cxGfR5lZ9DQs0wW\nAhbv2M+T3Zt9wL2yF2IzLJeapoAQ3i4YJSt7P5kVqetTfsrXlREms7GxP6z+5xTt\nIAb3M88vVEjb82ITQh+5V7P6JfxVXZJWwU0zozKmnojo0oyBVnJabbnL6eikmh24\njRCoMbt+IivN8GU8hhUeo45jOKz66oJP6mOteAIvMchOj3vHQA8eFAJb+8cnUJIJ\n4ubvpFDjaH5Zqd3rGX4+1HXHk3rOFgaBavDm9wyJQvERA3Gfy4Jt3xK7tCcBf/2v\njAzAg94raFT4BJ4AsWD7uvuK4BXKkRGjJ6oggsVA9hnqbOqiHwZ7kkeRjYKQIrZ2\nuuWQ7sMqWY4k6RCLVvYI4K1Jj70qw1HJ+rjmZFLoUI1BAgMBAAGjgZ8wgZwwDwYD\nVR0TAQH/BAUwAwEB/zAOBgNVHQ8BAf8EBAMCAQYwHQYDVR0OBBYEFEstCtIsRctQ\nYHONaewCyWr3SHSsMFoGA1UdIwRTMFGAFEstCtIsRctQYHONaewCyWr3SHSsoS+k\nLTArMSkwJwYDVQQDDCBib290c3RyYXAtMC5sb2NhbC52aXJ0dWFsYm94Lm1hY4II\nbubuNzD0ysgwDQYJKoZIhvcNAQELBQADggIBAJ/+20BB51qRH0ImEp52BIUWg7kt\nmrelyZiRCQ0YvZ4CsnLs+YOcz2lwh4GK8MEBaYpRAJHqcS2YnvQBd4S9JLsEbWMw\nwZnxUpMS8cMeXehjSPsh0VP13R76CB5NFlI0XY9QIR9JCZQpXZrnKXm/qil6Go6B\ngsd2qSKsmg5JZA71HI0ZwwkAkNZAa2u5c9bXn3X6vHoBjxZl0xnS6QElhNPeUc40\nIfX5BVJPHL51D2f0s8pfFlH1MGucmD6rGX4PG86OLW7NMr+ohE7E2d2opsr3fdsQ\nrmLrH4u3g124f63ApwqWMVRGhzGktCBI2i8HqT5GkJIfA7alhJETv/EJVC21ECJC\nza/1AHfYffqzt/wq6hVgqXAaGZQePX4LFw8HkGj2qSVdPkOc+QQg8WoVakFLUMBG\nd5tp28ZdZvmXPg3mwmeDrsAZKrvgvitNAP42yWUIAB0uF+63JcJoJQmnsR4hC0zx\njHts6R61163ZBPqypyqz6gkPGZJlAXtdgXDer2DsK92jTeuCO/RLbRg4xKT2afTn\nTdJ4LnAshpdsm5tTW/7K7nH1pnbVUdWJA+gg//kc28YNL1kYz2YvU19ppf9jCMAH\n2a0VSRGBA8B2BibqLqmcUMlNvTUhSJ3K2hpPrVBX+QrAfUpLziETONzGr+/BA8Bd\nLvEU38kGxPGSgEJ6\n-----END CERTIFICATE-----\n'
```

So it should probably be like:
```
contents += str(append_file_contents.decode('utf-8'))
```
to ensure it is properly formatted.